PhpAmqpLib/Wire/IO/StreamIO.php

```diff
@@ -247,9 +247,14 @@ public function read($len)
     public function write($data)
     {
         $written = 0;
+        $buffer  = false;
+        $tw      = false;
```

Comment:
What does `$tw` stand for/mean?  Can you make the variable name more descriptive of what it is?  Trying to move the library towards better variable naming.  Thanks!